src/zoauthServer.js

```diff
@@ -479,7 +479,7 @@ export class ZOAuthServer {
 
       return { result };
     } catch (error) {
-      return { result: { error: error.message } };
+      return { result: { error: error.message, status: error.status } };
```

Comment:
Too bad we still need to return error manually on main services catch ...
Cf #46